commons/zenoh-codec/src/bin/no_std_ci_check.rs

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright (c) 2022 ZettaScale Technology
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+//
+// Contributors:
+//   ZettaScale Zenoh Team, <zenoh@zettascale.tech>
+//
+
+#![no_std]
+
+#[panic_handler]
+fn dummy_panic_handler(_: &core::panic::PanicInfo) -> ! {
```

Comment:
This doesn't pass `cargo check --all-features` because std defines the panic handler. Add `#[cfg(not(feature="std"))]` to fix it :)